website/static/js/registrationEditorExtensions.js

```diff
@@ -180,59 +181,45 @@ var Uploader = function(data) {
 
 var AuthorImport = function(data, $root) {
     var self = this;
-
     self.question = data;
-    self.contributors = $root.contributors();
 
-    self.setContributorBoxes = function(value) {
-        var boxes = document.querySelectorAll('#contribBoxes input[type="checkbox"]');
-        $.each(boxes, function(i, box) {
-            this.checked = value;
-        });
+    self.makeContributorsRequest = function() {
+        var self = this;
+        var contributorsUrl = window.contextVars.node.urls.api + 'get_contributors/';
+        return $.getJSON(contributorsUrl);
+    };
+    self.getContributors = function() {
+        var self = this;
+        return self.makeContributorsRequest()
+            .then(function(data) {
+                return $.map(
```

Comment:
Very minor, but can you move this mapping function (used below as well), to a shared function?